refresh/templates/common/README.scaffold.md

```diff
@@ -65,6 +58,70 @@ This application has been generated with the following capabilities and services
 * [Auto-scaling](#auto-scaling)
 <% } -%>
 
+
+### Requirements
+* [Swift 4](https://swift.org/download/)
+
+### Run
+To build and run the application:
+1. `swift build`
+1. `.build/debug/<%- executableName %>`
+
+<% if (docker) { -%>
+#### Docker
+A description of the files related to Docker can be found in the [Docker files](#docker-files) setion. To build the two docker images, run the following commands from the root directory of the project:
+* `docker build -t myapp-run .`
+* `docker build -t myapp-build -f Dockerfile-tools .`
+You may customize the names of these images by specifying a different value after the `-t` option.
+
+To compile the application using the tools docker image, run:
+* `docker run -v $PWD:/swift-project -w /swift-project myapp-build /swift-utils/tools-utils.sh build release`
+
+To run the application:
+* `docker run -it -p 8080:8080 -v $PWD:/swift-project -w /swift-project myapp-run sh -c .build-ubuntu/release/<%- executableName %>`
+
+
+#### Kubernetes
+To deploy your application to your Kubernetes cluster, run `helm install --name myapp .` in the `/chart/<%- chartName %>` directory. You need to make sure you change the `repository` variable in your `chart/<%- chartName %>/values.yaml` file points to the docker image containing your runnable application.
```

Comment:
"You need to make sure you change the `repository` variable in your `chart/<%- chartName %>/values.yaml` file points to the docker image containing your runnable application." doesn't read correctly. How about instead of "file points" "to point to the"